src/lib/items.js

```diff
@@ -0,0 +1,16 @@
+import useFetch from "./fetch";
+import { useState } from "react";
+
+export default function useItems(url) {
+  const [URL, setURL] = useState(url);
+  const items = useFetch(URL);
+
+  function handleNextPage() {
```

Comment:
Die Funktion handleNextPage und handlePreviousPage kommen doch eh nur einmal in der ItemList.js vor oder?
Dann hätte man ja nicht wirklich was gewonnen außer eine aufgeräumtere ItemList.js

Reply:
In dem Falle könnte mans auch lassen, aber ich dachte es ist eine gute Fingerübung. Außerdem bin ich dem Dogma der React-Docs gefolgt: `When you extract logic into custom Hooks, you can hide the gnarly details of how you deal with some external system or a browser API. The code of your components expresses your intent, not the implementation.` s. hier: https://beta.reactjs.org/learn/reusing-logic-with-custom-hooks